lib/fastlane/plugin/ionic/actions/ionic_action.rb

```diff
@@ -137,7 +138,12 @@ def self.set_build_paths(is_release)
         app_name = self.get_app_name
         build_type = is_release ? 'release' : 'debug'
 
-        ENV['CORDOVA_ANDROID_RELEASE_BUILD_PATH'] = "./platforms/android/app/build/outputs/apk/#{build_type}/app-#{build_type}.apk"
+        # Update the build path accordingly if Android is being
+        # built as an Android Application Bundle.
+        android_package_type = params[:android_package_type || 'apk'
```

Comment:
Looks like we got a slight syntax error here 😊 

```suggestion
        android_package_type = params[:android_package_type] || 'apk'
```